br/pkg/lightning/config/const.go

```diff
@@ -34,3 +38,11 @@ const (
 
 	DefaultBatchSize ByteSize = 100 * units.GiB
 )
+
+var (
+	DefaultGrpcKeepaliveParams = grpc.WithKeepaliveParams(keepalive.ClientParameters{
+		Time:                10 * time.Second,
+		Timeout:             3 * time.Second,
```

Comment:
It seems `Timeout` should be larger than `Time`. Otherwise, `Time` is useless.

Reply:
```
	// After a duration of this time if the client doesn't see any activity it
	// pings the server to see if the transport is still alive.
	// If set below 10s, a minimum value of 10s will be used instead.
	Time time.Duration // The current default value is infinity.
	// After having pinged for keepalive check, the client waits for a duration
	// of Timeout and if no activity is seen even after that the connection is
	// closed.
	Timeout time.Duration // The current default value is 20 seconds.
```
According to this comment, I think they are irrelevant. I change this to 20 seconds which is the default value.

Reply:
I remember these values are copied from CDC? maybe they have the reason to choose it

Reply:
very old commit 😂  https://github.com/pingcap/tiflow/commit/9270a7cfdf9fde2d771c0a8b45fe914dab5ab007#diff-31a8a1f4c2bd92516aebe3c694c8ab06eb69f88aa09a95212f1f5e81a91fcf1c

Reply:
I think that doesn't matter much. Lightning needn't to assure second-level replication. So I think cutting this limit a bit loose is reasonable.

Reply:
From my understanding, if `timeout` is less than `time`,  the connection is always closed before it can ping the server when there is no activity.